WordPress/Classes/ViewRelated/Notifications/Views/LikeUserTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+class LikeUserTableViewCell: UITableViewCell, NibReusable {
+
+    // MARK: - Properties
+
+    @IBOutlet weak var gravatarImageView: CircularImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var usernameLabel: UILabel!
+    @IBOutlet weak var separatorView: UIView!
+    @IBOutlet weak var separatorHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var separatorLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet weak var cellStackViewLeadingConstraint: NSLayoutConstraint!
+
+    static let estimatedRowHeight: CGFloat = 80
+    private typealias Style = WPStyleGuide.Notifications
+
+    // MARK: - View
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureCell()
+    }
+
+    // MARK: - Public Methods
+
+    func configure(withUser user: RemoteUser, isLastRow: Bool = false) {
+        nameLabel.text = user.displayName
+        usernameLabel.text = String(format: Constants.usernameFormat, user.username)
+        downloadGravatarWithURL(user.avatarURL)
+        separatorLeadingConstraint.constant = isLastRow ? 0 : cellStackViewLeadingConstraint.constant
+    }
+
+}
+
+// MARK: - Private Extension
+
+private extension LikeUserTableViewCell {
+
+    func configureCell() {
+        nameLabel.textColor = Style.blockTextColor
+        usernameLabel.textColor = .textSubtle
+        backgroundColor = Style.blockBackgroundColor
+        separatorView.backgroundColor = Style.blockSeparatorColor
+        separatorHeightConstraint.constant = .hairlineBorderWidth
+    }
+
+    func downloadGravatarWithURL(_ url: String?) {
+        // Always reset gravatar
+        gravatarImageView.image = .gravatarPlaceholderImage
```

Comment:
Maybe also call `gravatarImageView.cancelImageDownload()` just in case the cell has been reused and there is already an inflight download?

Reply:
Oh good idea. Will do! Thanks!